product_input_interface/allowed_products.py

```diff
@@ -0,0 +1,28 @@
+class WarehouseProducts:
+    __ALLOWED_PRODUCTS_TYPE = ["helmet", "rifle", "medkit"]
+    __MAX_PRODUCT_NAME_LENGTH = 60
+    __MIN_PRODUCT_NAME_LENGTH = 1
+
+    def print_allowed_product_types(self):
+        print("Allowed product types:")
+        for item in self.__ALLOWED_PRODUCTS_TYPE:
+            print(f"- {item}")
+
+    def check_product_type(self, product_type: str):
```

Comment:
we have type hint here `product_type: str` - usually in such case it's also good to provide type hint for function's returned value

I would also rename that function a little to make it more self-describing e.g. `is_product_type_in_allowed_list`